cypress/e2e/supervision/clients/events/events.cy.js

```diff
@@ -0,0 +1,33 @@
+beforeEach(() => {
+  cy.loginAs("Case Manager");
+  cy.createClient();
+  cy.get("@client").then(({id}) => {
+    cy.visit(`/supervision/#/clients/${id}`);
+  });
+});
+
+describe(
+  "Events",
+  {
+    tags: ["@supervision-core", "@client-risk", "@smoke-journey"],
+  },
+  () => {
+    it("Successfully create a client event in Supervision", () => {
+      cy.get('#create-event-button').click();
+      cy.get('*[id^="FIELDLABELSN_"]').first().select(1);
+      cy.get('.footer > :nth-child(1) > .button').click();
+      cy.get('.TABS_TIMELINELIST').click();
+      cy.get('.event-note > .section-content > .wrapper')
+        .should("contain", "event has been recorded")
+        .should("contain", "Event date");
+    });
+
+    it("Error create a client event in Supervision", () => {
```

Comment:
These validation summary tests should be moved down to Jest (or at least that's what we have been doing on the other Cypress tickets)

Reply:
I am not sure as some tickets say to do that and others dont, but it makes sense to move it down if thats what you want me to.

Reply:
changed to a jest